res/skins/QMLDemo/EffectSlot.qml

```diff
@@ -0,0 +1,196 @@
+import "." as Skin
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+import "Theme"
+
+Item {
+    id: root
+
+    property var slot: Mixxx.EffectsManager.getEffectSlot(1, unitNumber, effectNumber)
+    property int unitNumber // required
+    property int effectNumber // required
+    property bool expanded: false
+    readonly property string group: slot.group
```

Comment:
Didn't try, maybe the readonly flag from the slot property is also applying to the alias. I can check.

Reply:
Nope, doesn't work. Maybe because the order of property initializiation is not defined or something:

```
warning [Main] file:///home/jan/Projects/mixxx/res/skins/QMLDemo/main.qml:120:9: Type EffectRow unavailable
             EffectRow {
             ^
warning [Main] file:///home/jan/Projects/mixxx/res/skins/QMLDemo/EffectRow.qml:14:9: Type Skin.EffectUnit unavailable
             Skin.EffectUnit {
             ^
warning [Main] file:///home/jan/Projects/mixxx/res/skins/QMLDemo/EffectUnit.qml:24:9: Type EffectSlot unavailable
             EffectSlot {
             ^
warning [Main] file:///home/jan/Projects/mixxx/res/skins/QMLDemo/EffectSlot.qml:13:27: Invalid alias reference. Unable to find id "slot"
         property alias group: slot.group
                               ^
warning [Main] Skin "QMLDemo" failed to load!
```

Reply:
Mhmm thats strange. I guess I'm misinterpreting the Qt docs section https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#considerations-for-property-aliases